packages/office-ui-fabric-react/src/components/Announced/examples/Announced.QuickActions.Example.tsx

```diff
@@ -92,6 +96,18 @@ export class AnnouncedQuickActionsExample extends React.Component<{}, IAnnounced
     };
   }
 
+  public componentDidUpdate(prevState: IAnnouncedQuickActionsExampleState) {
+    if (prevState.announced !== this.state.announced && this.state.announced !== undefined) {
+      const interval1 = this._async.setInterval(() => {
```

Comment:
Spoke offline, prefer `setTimeout` here and clean-up in `componentWillUnmount` just in-case